docs/maps/maps-aggregations.asciidoc

```diff
@@ -39,12 +41,22 @@ The *Grid aggregation* source uses {ref}/search-aggregations-bucket-geotilegrid-
 
 You can symbolize grid aggregation metrics as:
 
+*Clusters*:: Creates a <<vector-layer, vector layer>> with a cluster symbol for each gridded cell.
+The cluster location is the weighted centroid for all geo-points in the gridded cell.
+
 *Grid rectangles*:: Creates a <<vector-layer, vector layer>> with a bounding box polygon for each gridded cell.
 
 *Heat map*:: Creates a <<heatmap-layer, heat map layer>> that clusters the weighted centroids for each gridded cell.
 
-*Clusters*:: Creates a <<vector-layer, vector layer>> with a cluster symbol for each gridded cell.
-The cluster location is the weighted centroid for all geo-points in the gridded cell.
+To enable grid aggregation:
+
+. Click *Add layer* button and select *Grid aggregation* source.
+
+To enable a blended layer that dynamically shows clusters or documents:
+
+. Click *Add layer* button and select *Documents* source.
+. Configure *Index pattern* and *Geospatial field*. To enable clustering, *Geospatial field* must be set to a field mapped as {ref}/geo-point.html[geo_point].
+. In *Scaling*, select the *Show clusters when results exceed 10000*.
```

Comment:
`select the *Show clusters when results exceed 10000*` > `select *Show clusters when results exceed 10000*`